src/sentry/event_manager.py

```diff
@@ -845,7 +845,7 @@ def save(self, project_id, raw=False, assume_normalized=False):
             tagstore.delay_index_event_tags(
                 organization_id=project.organization_id,
                 project_id=project.id,
-                group_id=group.id,
+                group_id=group.id if group else None,
```

Comment:
For issueless events we should not have non-Snuba event at all (even if we are not yet fully there), so by the time we even get to a prototype this should not be a problem. 
The plan to remove non Snuba implementation started here :
https://github.com/getsentry/sentry/pull/13700
in the sense that at least we are querying snuba for events. The further parts of the plan are:
- Remove all the explicit Event.objects.whatever (there are 10-20 of them. Mostly in corner cases)
- Fix all the tests that create event by writing to postgres instead of using the store_event flow
- Address the tagstore implementation
- Clean up the code and remove the feature that allows to skip snuba